rest-api/offline/public_metrics_export.py

```diff
@@ -0,0 +1,145 @@
+from sqlalchemy import text
+from dao import database_factory
+from dao.database_utils import replace_years_old
+from participant_enums import EnrollmentStatus, OrderStatus, PhysicalMeasurementsStatus
+from participant_enums import Race, QuestionnaireStatus
+
+# TODO(calbach): consider whether we need to filter down to consented
+# individuals only.
+
+def _questionnaire_metric(name, col):
+  """Returns a metrics SQL aggregation tuple for the given key/column."""
+  return (
+      name,
+      """
+      SELECT {}, SUM(1)
```

Comment:
Can we use count(*) instead of SUM(1)?

GROUP BY 1 also confused me initially... normally I'd expect GROUP BY <FIELD>. But I guess either works.